applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -126,6 +126,7 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_H) # Required for modify_distance_process
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VELOCITY) # Required for modify_distance_process
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PRESSURE) # Required for modify_distance_process
+        self.main_model_part.AddNodalSolutionStepVariable(KCPFApp.LIFT_COEFFICIENT)
```

Comment:
why does this have to be historical?

Reply:
It doesnt need to be historical, @BasselSaridar could you remove it?